.pre-commit-config.yaml

```diff
@@ -1,20 +1,20 @@
-# # Install pre-commit hooks via
-# pre-commit install
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-yaml
+      - id: check-json
+      - id: trailing-whitespace
+        exclude: miscellaneous/structures
```

Comment:
You probably want to exclude markdown files as well, as trailing spaces are valid syntax (double trailing spaces creates a new line, but not a new section).

Reply:
I'd consider that exotic enough to simply not allow unless needed.

Reply:
Ah, I use it often though :) But fair enough.

Reply:
Ok, then let's leave _as is_.